public/index.html

```diff
@@ -93,148 +93,169 @@
-                        <a class="thumbnail" href="#">
-                            <img id="nebula-22" class="img-responsive" src="" alt="">
-                        </a>
-                    </div>
-                    <div class="col-lg-3 col-centered thumb">
-                        <a class="thumbnail" href="#">
-                            <img id="nebula-23" class="img-responsive" src="" alt="">
-                        </a>
-                    </div>
-                    <div class="col-lg-3 col-centered thumb">
-                        <a class="thumbnail" href="#">
-                            <img id="nebula-24" class="img-responsive" src="" alt="">
-                        </a>
-                    </div>
-                </div>
-                <div class="row-centered">
-                    <div class="col-lg-3 col-centered thumb">
-                        <a class="thumbnail" href="#">
-                            <img id="nebula-19" class="img-responsive" src="" alt="">
-                        </a>
-                    </div>
-                    <div class="col-lg-3 col-centered thumb">
-                        <a class="thumbnail" href="#">
-                            <img id="nebula-6" class="img-responsive" src="" alt="">
-                        </a>
-                    </div>
-                    <div class="col-lg-3 col-centered thumb">
-                        <a class="thumbnail" href="#">
-                            <img id="nebula-7" class="img-responsive" src="" alt="">
-                        </a>
-                    </div>
-                    <div class="col-lg-3 col-centered thumb">
-                        <a class="thumbnail" href="#">
-                            <img id="nebula-8" class="img-responsive" src="" alt="">
-                        </a>
-                    </div>
-                    <div class="col-lg-3 col-centered thumb">
-                        <a class="thumbnail" href="#">
-                            <img id="nebula-9" class="img-responsive" src="" alt="">
-                        </a>
-                    </div>
+                <div class="col-lg-7">
+                    <div class="row plotbox" id="neighbourpca"></div>
                 </div>
-                <div class="row-centered">
-                    <div class="col-lg-3 col-centered thumb">
-                        <a class="thumbnail" href="#">
-                            <img id="nebula-18" class="img-responsive" src="" alt="">
-                        </a>
-                    </div>
-                    <div class="col-lg-3 col-centered thumb">
-                        <a class="thumbnail" href="#">
-                            <img id="nebula-5" class="img-responsive" src="" alt="">
-                        </a>
+                <div class="col-lg-5 gallery">
+                    <div class="row-centered">
+                        <div class="col-lg-3 col-centered selected-image-display">
```

Comment:
Tell me a bit more about these CSS classes? ie what's the number in `col-lg-x`? Is there different sizes within the class large? And if so, doesn't it make more sense to have `col-size-x` for all kinds of columns? Also, I'd rather be more explicit in these names (lg->large).